### PR TITLE
Seismic2firmware

### PR DESCRIPTION
**Pull request by salichon**
Combined Seismic (only CUSP3D)  GNSS ``install/firmware.csv`` files
Create issues in go routines
Not quite right yet I reckon ( go routines are not adapted)

### CODE REVIEW

**Comment by elidana** [comment]
nice!!  Thanks Jerome! 
A general comment (besides the checks failure)...I would personally remove the site code from your comment line, as this can be derived from the serial number and installation date in other files within the /install.

So instead of using 
` Fix GPS timing 1 sec STAS`
just use
` Fix GPS timing 1 sec` (same for all entries)?

**Comment by salichon** [comment]
@elidana Roger that
 for next iteration  :)

**Comment by salichon** [comment]
@ozym @elidana 
following your remarks and work 
the firmware.csv files  passed the test and is ready for  your review
thanks a lot 
cheers
jerome

**Comment by salichon** [comment]
Hello @ozym Ae you happy with that PR for firmware ? 
thanks  alot 
cheers
 
Jerome

File: install/firmware.csv [hunk 1]
```diff
@@ -2,6 +2,166 @@ Make,Model,Serial,Version,Start Date,End Date,Notes
...
+CSI,CUSP3D,42227,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42228,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42229,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42230,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42231,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42232,Cusp3D+r 400.74.1,2019-08-29T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42233,Cusp3D+r 400.74.1,2019-08-29T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42235,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42236,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42237,unknown,2014-11-27T00:00:05Z,2019-09-10T00:00:00Z,Timing error WAKS
```
Comment (elidana) [inline install/firmware.csv]: please remove site code from here

File: install/firmware.csv [hunk 2]
```diff
@@ -2,6 +2,166 @@ Make,Model,Serial,Version,Start Date,End Date,Notes
...
+CSI,CUSP3D,42253,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42255,Cusp3D+r 400.74.1,2019-08-29T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42256,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42257,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42258,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42259,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42260,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42261,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42262,Cusp3D+r 400.74.1,2019-08-16T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42263,Cusp3D+r 400.67,2015-04-21T23:51:30Z,2019-06-18T00:00:00Z,Timing error RDCS Sent to CSI
```
Comment (elidana) [inline install/firmware.csv]: please remove site code

File: install/firmware.csv [hunk 3]
```diff
@@ -2,6 +2,166 @@ Make,Model,Serial,Version,Start Date,End Date,Notes
...
+CSI,CUSP3D,42124,Cusp3D+r 400.74.1,2019-08-16T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42147,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42148,Cusp3D+r 400.74.1,2019-08-30T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42149,Cusp3D+r 400.74.1,2019-08-29T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42150,Cusp3D+r 400.74.1,2019-08-16T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42151,Cusp3D+r 400.74.1,2019-08-16T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42152,Cusp3D+r 400.74.1,2019-08-16T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42153,Cusp3D+r 400.74.1,2019-08-16T00:00:00Z,9999-01-01T00:00:00Z,Fix GPS timing 1 sec
+CSI,CUSP3D,42154,unknown,2014-07-15T22:20:00Z,2019-05-23T12:30:00Z,Timing error FGPS Sent to CSI
+CSI,CUSP3D,42155,Cusp3D+r 400.63,2014-08-06T04:12:00Z,2019-07-08T01:20:00Z,Timing error HWHS
```
Comment (elidana) [inline install/firmware.csv]: please remove site code

**Comment by salichon** [comment]
@elidana Done for your liking ;)

**Pull request merged**